src/components/JobCard/index.jsx

```diff
@@ -0,0 +1,19 @@
+function JobCard({ salary, type, title, company, logo, skills }) {
+  return (
+    <div className='flex py-5 px-2 justify-between items-start'>
```

Comment:
You're not allowed to have a div inside a TR, because some dudes decided that years ago.

It seems like you aren't actually really using the table. You've made a table that contains these JobCards, but then each JobCard essentially disregards the table and controls it's own layout.

You should go with one or the other here - either use a table entirely (converting the HTML in here to make use of <td>), or ditch the table and create headings using divs like you've done here.

Reply:
Changed to divs